Makefile

```diff
@@ -0,0 +1,20 @@
+MOCHA := "mocha"
+NYC := "nyc"
+
+TESTS = test/*.js
+TESTS_WITH_TIZEN = test/*.js libs/brackets-server/embedded-ext/project/test/*.js
+
+
+test:
+	@if ! type "tizen" > /dev/null; then make test-default; else make test-extension; fi
+
+
+test-default:
+	@echo "Run default test"
+	@$(NYC) --reporter=text --reporter=html $(MOCHA) --timeout=10000 $(TESTS)
```

Comment:
How about defining test_node to reduce duplicated code like as "https://github.com/mochajs/mocha/blob/master/Makefile"?

Reply:
Okay. Will remove duplicate code and introduce timeout as passed variable considering each test may need different timeout value.